spotbugs/src/doc/manual.xml

```diff
@@ -927,6 +927,18 @@ These options are only accepted by the Text User Interface.
   </listitem>
   </varlistentry>
 
+  <varlistentry>
+  <term><command>-parseStdinAsOptions</command> </term>
+  <listitem>
+    <para>
+    Read additional options from from standard input, with each new line as a separate argument.
+    This option is incompatible with other arguments using standard input,
+    such as -auxclasspathFromInput and reading files or data from standard input.
+    -auxclasspathFromInput can be replaced with -auxclasspath followed by a classpath line.
+    </para>
+  </listitem>
+  </varlistentry>
+
```

Comment:
Please update files under `/docs` directory too.

Reply:
@KengoTODA 

While `exclude` takes a single value, my `excludePath` is quite large.   Each module of the project has its own separately-maintained exclusions.

Note that this patch merely takes the approach that `auxclasspathFromInput` started and expands it to include other options.

A mcve is probably not reasonable, considering that large amounts of files are what triggers the situation.

You can find my ant task call here in the spotbugs/discuss project:

XTTE1020 error w/ html report; missing class "apply" #90 
https://github.com/spotbugs/discuss/issues/90

I will see about providing a sanitized example of what argument passing occurs in my project.

Reply:
Here is a heavily-sanitized version of the ant -v -d output with this patch in place.

```
 [spotbugs] Running SpotBugs...
Class java.util.UUID loaded from parent loader (parentFirst)
 [spotbugs] Executing '[...]/jdk1.8.0_60/jre/bin/java' with arguments:
 [spotbugs] '-Dfindbugs.hostApp=FBAntTask'
 [spotbugs] '-Dspotbugs.home=/home/user/java/spotbugs-4.0.1-SNAPSHOT'
 [spotbugs] '-classpath'
 [spotbugs] '/home/user/java/spotbugs-4.0.1-SNAPSHOT/lib/spotbugs.jar'
 [spotbugs] 'edu.umd.cs.findbugs.FindBugs2'
 [spotbugs] '-pluginList'
 [spotbugs] '/home/user/workspaces/xyz-oxygen/application-framework/building/lib-build/findsecbugs-plugin-1.4.6.jar'
 [spotbugs] '-effort:max'
 [spotbugs] '-sortByClass'
 [spotbugs] '-timestampNow'
 [spotbugs] '-html'
 [spotbugs] '-low'
 [spotbugs] '-parseStdinAsOptions'
 [spotbugs] '-parseStdinAsOptions'
 [spotbugs] '-parseStdinAsOptions'
 [spotbugs] '-outputFile'
 [spotbugs] '../deploying/dist/application-spotbugs-report-2020-03-11-11-15-16.html'
 [spotbugs] '-exitcode'
 [spotbugs] '/home/user/workspaces/xyz-oxygen/application-framework/WEB-INF/classes'
 [spotbugs] 
 [spotbugs] The ' characters around the executable and arguments are
 [spotbugs] not part of the command.
 [spotbugs] Using input "-exclude

 [spotbugs] -exclude
 [spotbugs] [...]/findbugs-[...]-exclude-filter.xml
**[REPEATS 10 times]**
 
 [spotbugs] -auxclasspath
 [spotbugs] **[36 ITEMS LISTED]**
 [spotbugs] -sourcepath
 [spotbugs] **[1442 ITEMS LISTED]**
 [spotbugs] "
Execute:Java13CommandLauncher: Executing '[...]/jdk1.8.0_60/jre/bin/java' with arguments:
'-Dfindbugs.hostApp=FBAntTask'
'-Dspotbugs.home=/home/user/java/spotbugs-4.0.1-SNAPSHOT'
'-classpath'
'/home/user/java/spotbugs-4.0.1-SNAPSHOT/lib/spotbugs.jar'
'edu.umd.cs.findbugs.FindBugs2'
'-pluginList'
'/home/user/workspaces/xyz-oxygen/application-framework/building/lib-build/findsecbugs-plugin-1.4.6.jar'
'-effort:max'
'-sortByClass'
'-timestampNow'
'-html'
'-low'
'-parseStdinAsOptions'
'-parseStdinAsOptions'
'-parseStdinAsOptions'
'-outputFile'
'../deploying/dist/application-spotbugs-report-2020-03-11-11-15-16.html'
'-exitcode'
'/home/user/workspaces/xyz-oxygen/application-framework/WEB-INF/classes'

The ' characters around the executable and arguments are
not part of the command.

```

Reply:
I noticed that `-parseStdinAsOptions` shows up once for each kind of option passed in which supports it.  For now there are only four, but I should fix it to output only once if we are going forward with this patch.